docs/source/kinds_of_types.rst

```diff
@@ -634,7 +634,7 @@ you pass it the right class object:
        # (Here we could write the user object to a database)
        return user
 
-How would we annotate this function?  Without ``Type[]`` the best we
+How would we annotate this function?  Without :py:class:`~typing.Type` the best we
```

Comment:
Is it wise to remove the brackets here? TBH I didn't understand their meaning here.